README.md

```diff
@@ -17,13 +17,13 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
-[] Undefined
-[] NaN
-[] Integers
-[] Arrays
-[] Null
+[x] Strings
+[x] Booleans
+[x] Undefined
```

Comment:
In this case, NaN is not considered a JS data type as well as an array, which might be considered more of a JS data structure or collection